src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -257,41 +257,36 @@ type VotingPeriod = Natural
 -- (quorum_threshold >= upvote + downvote) && (upvote > downvote)
 type QuorumThreshold = Natural
 
-emptyStorage :: Storage pm
-emptyStorage = Storage
-  { sLedger =  BigMap $ Map.empty
-  , sOperators = BigMap $ Map.empty
-  , sTokenAddress = genesisAddress
-  , sPendingOwner = Nothing
-  , sAdmin = genesisAddress
-  , sMigrationStatus = NotInMigration
-  , sVotingPeriod = 60 * 60 * 24 * 7 -- 7 days
-  , sQuorumThreshold = 4
-    -- ↑ any proposals that have less that 4 votes will be rejected
-    -- regardless of upvotes
-
-  , sProposals = BigMap $ Map.empty
-  , sProposalKeyListSortByDate = []
-  }
-
 mkStorage
-  :: Address
-  -> Map (Address, FA2.TokenId) Natural
-  -> Operators
+  :: "admin" :! Address
+  -> "votingPeriod" :? Natural
+  -> "quorumThreshold" :? Natural
   -> Storage pm
-mkStorage admin balances operators = Storage
-  { sLedger = BigMap balances
-  , sOperators = operators
+mkStorage admin votingPeriod quorumThreshold =
+  Storage
+  { sLedger = mempty
+  , sOperators = mempty
   , sTokenAddress = genesisAddress
-  , sPendingOwner = Nothing
-  , sAdmin = admin
+  , sPendingOwner = arg #admin admin
+  , sAdmin = arg #admin admin
   , sMigrationStatus = NotInMigration
-  , sVotingPeriod = 60 * 60 * 24 * 7 -- days
-  , sQuorumThreshold = 4
+  , sVotingPeriod = argDef #votingPeriod votingPeriodDef votingPeriod
+  , sQuorumThreshold = argDef #quorumThreshold quorumThresholdDef quorumThreshold
 
-  , sProposals = BigMap $ Map.empty
+  , sProposals = mempty
   , sProposalKeyListSortByDate = []
   }
+  where
+    votingPeriodDef = 60 * 60 * 24 * 7  -- 7 days
+    quorumThresholdDef = 4
+    -- ^ any proposals that have less that 4 votes (by default) will be rejected
+    -- regardless of upvotes
+
+emptyStorage :: Storage pm
```

Comment:
`emptyStorage` seems to be unused btw. Let's remove? It looks useless outside of tests because of `genesisAddress`. And even in tests it seems useless anyway because we don't have access to `genesisAddress` in Nettest.

Reply:
Yep, makes sense to me.